search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -63,9 +65,8 @@ function displayTopResultsForCurrentTrend(countryCode) {
 
   let countryData = topicData.countries
       .filter(countries => countries.country === countryCode);
-
   // Handle case where there are no results.
-  if (countryData.length === 0) {
+  if (countryData.length === 0 || countryData[0].averageSentiment == -500) {
```

Comment:
Is there a reason this is only ==, why not ===?

Reply:
That was an accident. I'll change it to === now.